packages/vuetify/src/components/VImg/VImg.tsx

```diff
@@ -288,33 +286,36 @@ export const VImg = genericComponent<VImgSlots>()({
       })
     }
 
-    useRender(() => (
-      <VResponsive
-        class={[
-          'v-img',
-          { 'v-img--booting': !isBooted.value },
-        ]}
-        style={{ width: convertToUnit(props.width === 'auto' ? naturalWidth.value : props.width) }}
-        aspectRatio={ aspectRatio.value }
-        aria-label={ props.alt }
-        role={ props.alt ? 'img' : undefined }
-        v-intersect={[{
-          handler: init,
-          options: props.options,
-        }, null, ['once']]}
-      >{{
-        additional: () => (
-          <>
-            <__image />
-            <__preloadImage />
-            <__gradient />
-            <__placeholder />
-            <__error />
-          </>
-        ),
-        default: slots.default,
-      }}</VResponsive>
-    ))
+    useRender(() => {
+      const [responsiveProps] = VResponsive.filterProps(props)
+      return (
+        <VResponsive
+          class={[
+            'v-img',
+            { 'v-img--booting': !isBooted.value },
+          ]}
+          { ...responsiveProps }
+          aspectRatio={ aspectRatio.value }
```

Comment:
does this need to be defined? it's handled in responsive props

Reply:
yes